qa/common/gen_qa_model_repository

```diff
@@ -49,6 +49,7 @@
 ############################################################################
 
 TRITON_VERSION=${TRITON_VERSION:=24.05}
+MODEL_TYPE=${MODEL_TYPE:="V100"}
```

Comment:
There should be a way to automatically extract compute capability in python script than defining a new env var.